interfaces/builtin/system_files.go

```diff
@@ -0,0 +1,71 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"fmt"
+	"strings"
+)
+
+const systemFilesSummary = `allows access to system files or directories`
+
+const systemFilesBaseDeclarationSlots = `
+  system-files:
+    allow-installation: false
+    deny-connection: true
```

Comment:
I see this error when running the spread tests.

+ snap install --edge core
error: cannot perform the following tasks:
- Mount snap "core" (5940) (installation not allowed by "system-files" slot rule of interface "system-files")

Reply:
Also, I see my test snap that it has not the system/personal files interface (I updated the interfaces just to be able to test them)
https://paste.ubuntu.com/p/WxJQY2Y7kB/

Reply:
The typical pattern is to not have `deny-connection: true` (see above).